src/entities/supportRequest.ts

```diff
@@ -99,16 +99,25 @@ export class SupportRequest extends BaseEntity {
     }
   }
 
-  static async getNextSupportRequest(): Promise<SupportRequest> {
+  static async getNextSupportRequest(supportType: SupportRequestType): Promise<SupportRequest> {
```

Comment:
We need to make this new param optional
```suggestion
  static async getNextSupportRequest(supportType?: SupportRequestType): Promise<SupportRequest> {
```